docs/en/manuals/bob.md

```diff
@@ -50,10 +51,17 @@ usage: bob [options] [commands]
  -pk,--private-key <arg>             Private key (Android)
  -r,--root <arg>                     Build root directory. Default is
                                      current directory
+    --strip-executable               Strip the dmengine of debug symbols
+                                     (when bundling iOS or Android)
  -tc,--texture-compression <arg>     Use texture compression as specified
-                                     in texture profiles (boolean)
```

Comment:
@sicher Has --texture-profiles been remove in the documentation manually? It does show up if you run the --help option.
If so, should we manually remove the --debug option?

Reply:
Not sure. I usually run with --help and copy the output. I think it's better to be in tune with what the output actually shows, even deprecated stuff.